wwwroot/data/regionMapping.json

```diff
@@ -307,13 +307,36 @@
             "regionProp": "ELECT_DIV",
             "textCodes": true,
             "aliases": [
-                "divisionnm",
-                "ced_aec",
                 "ced_aec_2011"
             ],
             "description": "Commonwealth electoral districts (Australian Electoral Commission)",
             "regionIdsFile": "data/regionids/region_map-FID_COM20111216_ELB_region_ELECT_DIV.json"
         },
+        "CED_AEC_2016": {
+            "layerName": "region_map:FID_COM20160509_ELB",
+            "server": "http://geoserver.nationalmap.nicta.com.au/region_map/ows",
+            "regionProp": "DIV_ID",
```

Comment:
Ok, so should we leave `ced_aec` associated with the 2011 boundaries for backward compatibility?  Or remove it entirely?